src/penn_chime/parameters.py

```diff
@@ -4,7 +4,7 @@
 `change_date`, so users can see when results have last changed
 """
 
-from numpy import log2  # type: ignore
+from numpy import log2, power  # type: ignore
```

Comment:
nit: I think we should `import numpy as np`

Totally nonblocking though

Reply:
Addressed by PR 208